src/main/java/com/wiley/autotest/selenium/elements/upgrade/v3/NullShouldImmediately.java

```diff
@@ -0,0 +1,60 @@
+package com.wiley.autotest.selenium.elements.upgrade.v3;
+
+import com.wiley.autotest.selenium.SeleniumHolder;
+import com.wiley.autotest.selenium.context.SearchStrategy;
+import com.wiley.autotest.selenium.elements.upgrade.TeasyElement;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+
+import java.util.function.Function;
+
+/**
+ * Created by vefimov on 26/04/2017.
```

Comment:
Is it the correct description?

Reply:
Technically yes 🤖 but I would prefer removing it